templates/emails/admin-cancelled-order.php

```diff
@@ -24,8 +24,8 @@
 */
 do_action( 'woocommerce_email_header', $email_heading, $email ); ?>
 
-<?php /* translators: %1$s: Customer full name. %2$s: Order number */ ?>
-<p><?php printf( esc_html__( 'Alas. Just to let you know &mdash; %1$s has cancelled order #%2$s:', 'woocommerce' ), esc_html( $order->get_formatted_billing_full_name() ), esc_html( $order->get_order_number() ) ); ?></p>
+<?php /* translators: %1$s: Order number, %2$s: Customer full name.  */ ?>
+<p><?php printf( esc_html__( 'Alas. Just to let you know &mdash; order #%1$s belonging to %2$s has been cancelled:', 'woocommerce' ), esc_html( esc_html( $order->get_order_number(), $order->get_formatted_billing_full_name() ) ) ); ?></p>
```

Comment:
Probably due to copy and paste error, `esc_html()` is being called inside another `esc_html()` call, resulting in a `PHP Warning:  printf(): Too few arguments` and an empty string. The same is happening in the other modified file.